language/diem-framework/modules/doc/AccountFreezing.md

```diff
@@ -256,8 +256,8 @@ The <code><a href="AccountFreezing.md#0x1_AccountFreezing_FreezingBit">FreezingB
 
 <pre><code><b>include</b> <a href="DiemTimestamp.md#0x1_DiemTimestamp_AbortsIfNotGenesis">DiemTimestamp::AbortsIfNotGenesis</a>;
 <b>include</b> <a href="CoreAddresses.md#0x1_CoreAddresses_AbortsIfNotDiemRoot">CoreAddresses::AbortsIfNotDiemRoot</a>{account: dr_account};
-<a name="0x1_AccountFreezing_addr$8"></a>
-<b>let</b> addr = <a href="Signer.md#0x1_Signer_spec_address_of">Signer::spec_address_of</a>(dr_account);
+<a name="0x1_AccountFreezing_addr$12"></a>
+<b>let</b> addr = <a href="_spec_address_of">Signer::spec_address_of</a>(dr_account);
```

Comment:
How this link was created is unclear to me, points to a bug.

Reply:
In case it helps: I was passed in "" as doc_path when generating this.